src/protocol/creation/routes.rs

```diff
@@ -89,6 +89,13 @@ pub async fn create_file(
         }
     }
 
+    if let Some(max_file_size) = state.config.max_file_size {
```

Comment:
Also, it can be rewritten a bit to make it look nicer. 

```rust 
if state.config.max_file_size.is_some() && state.config.max_file_size < length {
   ...
}
```

Reply:
done